docs/resources/frontend_o11y_app.md

```diff
@@ -0,0 +1,58 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_frontend_o11y_app Resource - terraform-provider-grafana"
+subcategory: "Frontend Observability"
+description: |-
+  
+---
+
+# grafana_frontend_o11y_app (Resource)
+
+
+
+## Example Usage
+
+```terraform
+data "grafana_cloud_stack" "teststack" {
+  provider    = grafana.cloud
+  name        = "gcloudstacktest"
+}
+
+resource "grafana_frontend_o11y_app" "test-app" {
+  provider        = grafana.cloud
+  stack_id        = data.grafana_cloud_stack.teststack.id
+  name            = "test-app"
+  allowed_origins = ["https://grafana.com"]
+
+  extra_log_attributes = {
+    "terraform" : "true"
+  }
+
+  settings = {
```

Comment:
I couldn't see any docs on what the possible settings are for this block - is that something we plan to add? or maybe a link to some other docs that indicate what options are available?

Reply:
I added documentation on the settings that we currently have available (More to be released soon ;) )